lib/jsbi.ts

```diff
@@ -1104,6 +1103,7 @@ class JSBI extends Array {
         case 2: return x > y;
         case 3: return x >= y;
       }
+      throw new Error('unreachable');
```

Comment:
Why is this needed? Can't you annotate {op} in this function as {0|1|2|3} too?

Reply:
Yes - Done.